### PR TITLE
Updates to Anomaly Detection

### PR DESCRIPTION
**Pull request by bstuder99**

### CODE REVIEW

File: using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md [hunk 1]
```diff
@@ -4,12 +4,23 @@
...
+    enabled: true
+```
+
 ## Managing anomalies
 
 Selecting an anomaly will open the Cloud Cost Explorer with a filter for that specific service, allowing you to observe more cost metrics for that service.
 
-ANomaly detection is determined by the provided window of time of data for Kubecost to sample (default *Last 7 days*). This range establishes how much acitvity Kubecost will sample per service to detect anomalies, and will only display anomalies detecting in that window.
\ No newline at end of file
+ANomaly detection is determined by the provided window of time of data for Kubecost to sample (default *Last 30 days*). This range establishes how much acitvity Kubecost will sample per service to detect anomalies, and will only display anomalies detected in that window.
```
Comment (jessegoodier) [inline using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md:26]: lower case N

File: using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md [hunk 2]
```diff
@@ -4,12 +4,23 @@
...
+Anomaly Detection requires at least one cloud provider integration in order to detect anomalies in service spend. See our [Cloud Integration Guide](/install-and-configure/install/cloud-integration/README.md) to get started.
+{% endhint %}
 
 ![Anomaly detection](/images/anomalydetection.png)
 
+The Forecasting Container can be disabled by setting the Helm flag:
+
+```
+forecasting:
+    enabled: true
```
Comment (jessegoodier) [inline using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md:19]: should just be two spaces and false:
```
  enabled: false
```

**Review by jessegoodier — DISMISSED** [review]

File: using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md (hunk 2 above)
Comment (bstuder99) [reply using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md:19]: What do you mean by two spaces?

Comment (patrick-tavares) [reply using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md]: I think he means the indentation of `enabled: false`... incorrect YAML indentation spacing can cause issues.

File: using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md [hunk 3]
```diff
@@ -4,12 +4,23 @@
...
+Anomaly Detection requires at least one cloud provider integration in order to detect anomalies in service spend. See our [Cloud Billing Integrations](/install-and-configure/install/cloud-integration/README.md) doc to get started.
+{% endhint %}
 
 ![Anomaly detection](/images/anomalydetection.png)
 
+The Forecasting Container can be disabled by setting the Helm flag:
+
+```
+forecasting:
+        enabled: false
```
Comment (srpomeroy) [inline using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md:19]: Should only be indented two spaces beyond `forecasting:`

```suggestion
  enabled: false

```

**Review by srpomeroy — CHANGES_REQUESTED** [review]

File: using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md (hunk 2 above)
Comment (srpomeroy) [reply using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md]: Added an update comment with suggested changes.

**Review by jessegoodier — APPROVED** [review]

**Review by srpomeroy — APPROVED** [review]
LGTM

**Pull request merged**